processing/data_collection/gazette/spiders/instar_base.py

```diff
@@ -0,0 +1,34 @@
+import datetime as dt
+from dateparser import parse
+from scrapy import Request
```

Comment:
Just a minor detail, but according to [PEP8's guide for imports](https://www.python.org/dev/peps/pep-0008/#imports) it is preferable to separate import into blocks like this:

>Imports should be grouped in the following order:
>
>Standard library imports.
>Related third party imports.
>Local application/library specific imports.
>You should put a blank line between each group of imports.